cc-raisely-components/components/facil-conversation-list/facil-conversation-list.js

```diff
@@ -32,16 +50,26 @@
 			if (!Event) Event = EventRef().html;
 			this.displayDate = Event.displayDate(conversation);
 
-			this.reconciled = cashDonationsNotes ||
-				((cashTransferAmount === cashReceivedAmount) && (cashTransferAmount === cashCtaAmount));
+			this.reconciled =
+				cashDonationsNotes ||
+				(cashTransferAmount === cashReceivedAmount &&
+					cashTransferAmount === cashCtaAmount);
 		}
 
+		markCheckbox = (value) => {
+			const { conversation, setSelectedConversations } = this.props;
+
+			this.setState({ checked: value });
```

Comment:
This isn't good react practice to store state in two different places.
If you're storing state in the component that's enclosing it, then (except in special cases) you should be deferring completely to that state.
So this function should remove this call, and checked should be passed in from the enclosing component

(That said, you can probably find an example of me doing this somewhere in the cc-raisely-components as I was very new to react when I first wrote most of this)